benefits/eligibility/views.py

```diff
@@ -35,16 +36,15 @@ def index(request):
 
     eligibility_start = reverse(ROUTE_START)
 
+    help_page = reverse(ROUTE_HELP)
+
     page = viewmodels.Page(
         title=_("eligibility.pages.index.title"),
         content_title=_("eligibility.pages.index.content_title"),
-        paragraphs=[_("eligibility.pages.index.p[0]%(info_link)s")],
+        paragraphs=[format_html(_("eligibility.pages.index.p[0]%(info_link)s") % {"info_link": help_page})],
```

Comment:
```suggestion
paragraphs=[format_html(_("eligibility.pages.index.p[0]%(info_link)s") % {"info_link": f"{help_page}#about"})],
```

I think we want to keep the anchor link in there